File: program/check-lower-case/check_lower_case.dart
void checkLower(String c){
  int ascii = c.codeUnitAt(0);
  if(ascii >= 97 && ascii <= 122){
    print("Character is in lower case.");
  }
  else{
    print("Character is not in lower case.");
  }
}

void main() {
  checkLower("s");
}

File: program/program/check-lower-case/check_lower_case.go
package main

import (
	"fmt"
	"unicode"
)

func main() {
	var char rune
	fmt.Scanf("%c", &char)
	if unicode.IsLower(char) {
		fmt.Printf("Character '%c' is lower case\n", char)
	} else {
		fmt.Printf("Character '%c' is not in lower case\n", char)
	}
}
